src/styles/vars.css

```diff
@@ -26,10 +26,10 @@
   --color-info-main: #5fddff;
   --color-info-light: #b7f0ff;
   --color-info-background: #effcff;
-  --color-warning-dark: #cd674e;
+  --color-warning-dark: #C04C32;
   --color-warning-main: #ff8061;
-  --color-warning-light: #ffb7a6;
-  --color-warning-background: #fff0ed;
+  --color-warning-light: #FFBC9F;
+  --color-warning-background: #FFF1E0;
```

Comment:
This file is auto-generated and will be overwritten. Instead you have to change the colors in `/src/components/theme/lightPalette.ts` and `/src/components/theme/darkPalette.ts` and run `yarn css-vars` to update this file.